PowerPointLabs/PowerPointLabs/Utils/Graphics.cs

```diff
@@ -79,15 +79,23 @@ public static void ExportShape(ShapeRange shapeRange, string exportPath)
                               slideHeight, PpExportMode.ppScaleToFit);
         }
 
-        public static Image ShapeToImage(Shape shape)
+        public static Bitmap ShapeToImage(Shape shape)

```

Comment:
Is there any other place in pptlabs that implements lock/sync that I should follow? Or should I just implement it using c# Lock keyword?